tests/scripts/run-throughput-perf.py

```diff
@@ -290,6 +295,11 @@ def runIterations(dll_name, dll_path, iterations, crossgen_path, jit_path, assem
     if opt_level == 'min_opt':
         my_env['COMPlus_JITMinOpts'] = '1'
 
+    if jit_name == 'legacy':
+        my_env['COMPLUS_AltJit'] = '*'
```

Comment:
That's all that I wanted to confirm. Using legacyjit.dll should produce faster throughput than x86 ryujit.